docs/sdkx-react-native/sdk-configuration.mdx

```diff
@@ -0,0 +1,303 @@
+---
+sidebar_position: 30
+title: SDK Configuration
+description: "Helpshift provides several config options which can be used to customize the behaviour of the SDK."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# SDK Configuration
+
+<Intro>
+
+Helpshift provides several config options which can be used to customize the behaviour of the SDK.
+
+</Intro>
+
+## Install call configurations
+
+### Enable Logging {#enableLogging}
+
+Upon setting `enableLogging` to `true`, Helpshift SDK logs will be generated in the Android logcat/ Xcode console. Turning on logging can help developers resolve common integration issues.
+
+<div className="compact ">
+
+|         |                                     |
+| ------- | ----------------------------------- |
+| Key     | `enableLogging` |
+| values  | true/false                          |
+| default | true                                |
```

Comment:
i dont think the default is true for enableLogging

Reply:
Done